### PR TITLE
feat: show (build) version number

### LINKED ISSUE #1329: Version/build info presentation on main application view
Would be nice to display build info on main application view near logo. Some information could be displayed after clicking on version.

### PR DESCRIPTION
**Pull request by JulianWielga**
closes #1329

Zrzut ekranu z 2021-09-21 13-58-47

### CODE REVIEW

**Review by mproch — COMMENTED** [review]
Let's change it to staging, not releae/1.0

File: ui/client/http/HttpService.ts [hunk 1]
```diff
@@ -39,6 +39,26 @@ export type FetchProcessQueryParams = Partial<{
 
 export type StatusesType = Record<ProcessType["name"], ProcessStateType>
 
+export interface AppBuildInfo {
+  "name": string,
+  "gitCommit": string,
+  "buildTime": string,
+  "version": string,
+  "processingType": {
```
Comment (dswiecki) [inline ui/client/http/HttpService.ts:47]: it's a map and can be dynamic

**Comment by dswiecki** [comment]
VersionInfo interferes with panels, we need to change it

**Comment by JulianWielga** [comment]
> VersionInfo interferes with panels, we need to change it

this is intentional, inspired by windows buildinfo. it is needed only occasionally, mainly by devs. it makes no sense to make it more exposed/visible. when you hover(visible part) makes it frontmost and visible. toolbars are collapsible.

File: ui/client/http/HttpService.ts [hunk 2]
```diff
@@ -39,6 +39,26 @@ export type FetchProcessQueryParams = Partial<{
 
 export type StatusesType = Record<ProcessType["name"], ProcessStateType>
 
+export interface AppBuildInfo {
```
[the remark this answers is not in the document]
Comment (JulianWielga) [reply ui/client/http/HttpService.ts:42]: better now?

**Comment by arkadius** [comment]
BUILD_VERSION -> NUSSKNACKER_VERSION

**Review by arkadius — APPROVED** [review]

**Review by dswiecki — APPROVED** [review]

**Pull request merged**